mk/serviceconfig.mk

```diff
@@ -28,6 +28,15 @@ configs/: .build-artefacts/last-version \
 	$(foreach topic, $(TOPICS), $(foreach lang, $(LANGS),curl -s --retry 3 --fail -o configs/${lang}/catalog.${topic}.json http:$(API_URL)/rest/services/$(topic)/CatalogServer?lang=$(lang) && jq . configs/${lang}/catalog.${topic}.json 1> /dev/null; ))
 
 
+meteoconfigs/: export METEO_TESTING_STYLE_BASEURL=$(METEO_STYLE_BASEURL)
+meteoconfigs/:  configs/
```

Comment:
what is achieved by declaring two times the same rule in a row? won't the last rule overwrite the first?